front/src/routes/dashboard/style.css

```
@@ -1204,6 +1204,64 @@
   }
 }
 
+/* The weather forecast rows scroll sideways on purpose — their slots keep a
+   readable floor rather than being crushed into a narrow card. What they must
+   not do is carry a permanent scrollbar across the card: it reads as a broken
+   layout, which is how it was reported. Same treatment as the app's other
+   sideways rows (.hz-tabs, .groupChips): the bar is hidden, and a thin one
+   comes back on hover or keyboard focus for mouse users. */
+:global(.glass-theme .weather-forecast-scroll) {
```

Stylelint (17.14.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)